### PR TITLE
Migrate travis ci to Github actions.

### PR DESCRIPTION
**Pull request by KaylaNguyen**
* Run CI tests against go 1.11 to 1.15 

* Run gopath test for go 1.11 and go 1.12

* Update README with CI status

### CODE REVIEW

**Review by joelgerard — APPROVED** [review]
Can you also update go/new-go-sdk and/or go/creating-a-new-gae-standard-runtime

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: off
+      run: |
+        go get -u -v $(go list -f '{{join .Imports "\n"}}{{"\n"}}{{join .TestImports "\n"}}' ./... | sort | uniq | grep -v appengine)
+    - name: Test gopath
+      shell: bash
```
Comment (matthewrobertson) [inline .github/workflows/ci.yml:50]: do you need to specify the shell here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: on
+      run: |
+        go get .
```
Comment (matthewrobertson) [inline .github/workflows/ci.yml:26]: not sure how much perf matters, but github actions supports caching go modules between builds for decreased build times: https://github.com/actions/cache/blob/main/examples.md#go---modules

**Review by matthewrobertson — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (KaylaNguyen) [reply .github/workflows/ci.yml:35]: Nice! Added.

**Pull request merged**